walletnode/api/services/sense.go

```diff
@@ -115,6 +119,67 @@ func (service *Sense) ActionDetails(ctx context.Context, p *sense.ActionDetailsP
 	return res, nil
 }
 
+// StartProcessing - Starts a processing image task
+func (service *Sense) StartProcessing(ctx context.Context, p *sense.StartProcessingPayload) (res *sense.StartProcessingResult, err error) {
+	ticket := &senseregister.Request{
+		BurnTxID:              p.BurnTxid,
+		AppPastelID:           p.AppPastelID,
+		AppPastelIDPassphrase: p.AppPastelidPassphrase,
+	}
+
+	// get image filename from storage based on image_id
+	filename, err := service.db.Get(p.ImageID)
```

Comment:
This is design layer & you are accessing db\storage layer from design layer which is not the way to go. 
Please review code of artwork register (or other APIs) & follow the same approach.

Reply:
db\storage layer and other services should be accessed from application layer.

Reply:
db  storage.KeyValue - in this layer to store image filename + ID. Same as artwork layer:
https://github.com/pastelnetwork/gonode/blob/9394a8bc59af9826d1ce6e8ea5369ac3028ce6f1/walletnode/api/services/artwork.go#L106